float64 angle
uint8 x_cordinate
uint8 y_cordinate
float64 velocity
string id